Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/FileBrowserHelper.java

```diff
@@ -29,11 +29,25 @@ public static void openDirectoryPicker(FragmentActivity activity) {
         activity.startActivityForResult(i, MainPresenter.REQUEST_ADD_DIRECTORY);
     }
 
+    public static void openFilePicker(FragmentActivity activity, int requestCode)
+    {
+        Intent i = new Intent(activity, CustomFilePickerActivity.class);
+
+        i.putExtra(FilePickerActivity.EXTRA_ALLOW_MULTIPLE, false);
```

Comment:
Does this mean that the user will be able to pick more than one file?

Reply:
if we pass true to the property we can allow the user to pick more than one file? do we need that?

Reply:
Oh, sorry, I missed that it said false. So this line doesn't need any changes, because there's not reason to pick more than one file.